packages/static-build/test/fixtures/dojo-v2/package.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
<!-- Requirement comments refer to a task requirements that is not being met by the current code. Mention what the requirement is and why the code does not meet it.  -->
**requirement (blocking):** please ‏add a lockfile